src/libs/ReactNativeLauncher.js

```diff
@@ -59,14 +60,34 @@ class ReactNativeLauncher extends Launcher {
     log.debug('bridges init done')
   }
 
-  async ensureConnectorIsInstalled({ connector, client }) {
-    await ConnectorInstaller.ensureConnectorIsInstalled({
-      ...connector,
-      client
-    })
-    const content = await ConnectorInstaller.getContentScript(connector)
-    const manifest = await ConnectorInstaller.getManifest(connector)
-    return { content, manifest }
+  async ensureConnectorIsInstalled({ slug, client }) {
+    try {
+      await updateCozyAppBundle({
+        slug,
+        client,
+        type: 'konnectors'
```

Comment:
nit: as this is type enforced, you can also declare `BUNDLE_TYPE_KONNECTOR` and `BUNDLE_TYPE_APP` consts in `cozyAppBundle.ts`

Reply:
I'm not too sold as there are only two values, typescript will forbid you to use anything else than what is expected. Now, this file is in js, but since I'm working on it believe me it won't stay like that for long 😂